app/generator/content/schedule/part2/get-reduction-zero-description.js

```diff
@@ -0,0 +1,11 @@
+const getReductionZeroDescription = (schedule) => {

```

Comment:
Just double checking this name is right.

Can we get a zero payment from a top-up rather than reduction?

E.g. PR1 is 1000 | Paid 250 | PR2 is 200 | PR3 is 250

Would this follow the same business path?

Reply:
The name is right, this ticket only covers path of reduction to zero from positive. The example you gave is reduction from negative (recovery) to zero, we would need to consider some AR entries to treat this scenario. Issues around getting AR data was raised during our last meeting, it was concluded that the scenario of recovery to zero is possible but very rare, hence we should treat case as reduction to zero for now.